pages/search/search.js

```diff
@@ -0,0 +1,109 @@
+import Tool from '../../utils/tool.js';
+Page({
+
+  /**
+   * 页面的初始数据
+   */
+  data: {
+    filterList: [],
+    searchText:''
+  },
+
+  /**
+   * 生命周期函数--监听页面加载
+   */
+  onLoad: function (options) {
+  },
+  //搜索监听事件
+  inputEvent: function (event) {
+    let that = this;
+    //查询函数防抖动
+    Tool.debounce(function () {
+      console.log(event.detail.value);
+      console.log(that);
+      that.searchRequest(event.detail.value);
+    }, 300)();
+  },
+  //搜索请求
+  searchRequest: function (value) {
+    let temp = this;
+    wx.request({
+      url: 'https://api.github.com/search/repositories',
```

Comment:
I think you may use the wrong API here. The requirement is to list organization repos, so the API to use should be this one: https://developer.github.com/v3/repos/#list-organization-repositories